server-base/src/main/java/org/apache/kylin/rest/service/CacheService.java

```diff
@@ -27,11 +27,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.InitializingBean;
+import org.springframework.cache.CacheManager;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 
-import net.sf.ehcache.CacheManager;
```

Comment:
I don't understand why chenge to another `org.springframework.cache.CacheManager.CacheManager`, could you please explain ?